opcua_tools/nodeset_generator.py

```diff
@@ -97,17 +97,23 @@ def create_header_xml(
 
     model_uri = namespaces[serialize_namespace]
     try:
-        model = next(model for model in models if model.model_uri == model_uri)
+        model = next(model for model in models if model["uri"] == model_uri)
     except StopIteration:
         model = ua_models.UAModel(

```

Comment:
If you define the `model` variable in the new way, I think this one is redundant.

Reply:
I don't understand what you're suggesting. I've tried to not change any logic. I don't actually know why this line is written like this, it's a very uncommon pattern in Python. But, if it works I don't want to change it because I don't understand why it was written this way.